.github/workflows/release.yaml

```diff
@@ -0,0 +1,29 @@
+name: Releases
+
+on:
+  push:
+    branches: [ main ]
+
+jobs:
+  changelog:
+    runs-on: ubuntu-latest
```

Comment:
where does it run exactly - ubuntu?

Reply:
Github Action runs then internally.

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Tags:** Efficient